src/pages/HomePage.css

```diff
@@ -0,0 +1,44 @@
+body {
```

Comment:
All the CSS here should be contained within `.HomePage` to stop it from colliding with other things on the site. If you do this, you also need to add a `className` below.

An alternative solution would be to use the CSS in JavaScript technique like in some of the other components. That automatically stops collisions. I am open to either method, though. Using plain CSS is more approachable somehow.